AmericanMedicalAssociation/ama-style-guide-2
# [EWL-4610] SG2 | Create Agenda Table Molecule

## Ticket(s)

**Jira Ticket**
- [EWL-4610: SG2 | Create Agenda Table Molecule](https://issues.ama-assn.org/browse/EWL-4610)

## Description
As a user I want to see what time a session begins and the name and details of the session.
When I click "Molecules" in the menu, then I should see "Agenda Table". This table should have two columns with borders on the top of the rows. The date should looks like an h5 on the left side and on the right a h3 styled text for the agenda item title and, optionally, a body below.

This update creates the twig template, descriptive markdown, and relevant scss required to make the agenda table along with the backstop test case and reference images.

## To Test

- Pull `feature/EWL-4608-create-agenda-table-molecule` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Molecule > Agenda Table](http://localhost:3000/?p=agenda-table-molecule).
- Confirm molecule looks as shown in Zeplin mock up.
- Click on the gear icon in the top right corner of the screen and then click "show pattern info". relevant information about the agenda table molecule should appear.

## Visual Regressions

Reference images for the event page backstop tests have been included in this PR.


## Relevant Screenshots/GIFs
![image](https://user-images.githubusercontent.com/4438120/40085041-d3b8443e-585e-11e8-9ff4-a1e87636207e.png)


## Remaining Tasks
- [x] Update glossary to include agenda table definition.


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
